src/components/Table/components/Row.tsx

```diff
@@ -372,4 +379,6 @@ export type GridDataRow<R extends Kinded> = {
   selectable?: false;
   /** Whether this row should infer its selected state based on its children's selected state */
   inferSelectedState?: false;
+  /** Whether this row is draggable, usually to allow drag & drop reordering of rows */
+  draggable?: true;
```

Comment:
**nit:** Probably worth making this a boolean otherwise TS will sometimes require us to do `draggable: true as const`. If we allow it to be a boolean value, then we don't need the `as const` suffix.